docs/source/components.rst

```diff
@@ -240,6 +261,12 @@ Cross field validation is made possible by overriding ``Form.clean()``:
             return data
 
 
+If you use a form's cleaned data to create a new model then model validation
+will also take place. It can be customized by passing a list of validators to a
```

Comment:
I meant something like this:

```python
def my_view(request):
    form = MyForm(request.POST)
    if form.is_valid():
        MyModel.objects.create(foo=form.cleaned_data['foo'])
```
I intentionally didn't mention model forms because I wanted to emphasize that the validation is done by the model itself and not by the model form.